bin/turnt

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env node
+'use strict';
+
+const Crypto = require('crypto');
+const FS = require('fs');
+const HTTP = require('http');
+const HTTPS = require('https');
+const Path = require('path');
+const Signature = require('../lib/signature');
+const URL = require('url');
+
+global.Config = require('nconf');
+
+Config.argv({
+  config: {
+    alias: 'c',
+    describe: 'Path to local turnstile configuration'
+  },
+  method: {
+    alias: 'X',
+    describe: 'HTTP request method',
+    default: 'GET'
+  },
+  payload: {
+    alias: 'd',
+    describe: 'HTTP request payload',
+    default: false
+  },
+  digest: {
+    describe: 'Digest signing scheme',
+    default: 'sha256'
+  },
+  header: {
+    alias: 'H',
+    array: true,
+    describe: 'HTTP request headers'
+  },
+  output: {
+    alias: 'o',
+    describe: 'Write output to a file',
+    default: false
+  },
+
+  identity: {
+    alias: 'u',
+    demand: true,
+    describe: 'Identity key for the request'
+  },
+  secret: {
+    alias: 'p',
+    demand: true,
+    describe: 'Secret key for the request'
+  }
+});
+
+Config.defaults({
+  log: {
+    json: false,
+    timestamp: false,
+    colorize: true
+  }
+  // log: {
```

Comment:
Did you want to leave these commented out options here when they are specified in lib/config.js?